config/vufind/config.ini

```diff
@@ -2246,3 +2246,10 @@ show_solr_options_in_tag_search = false
 description = "The REST API provides access to search functions and records contained in the search index."
 ; URL pointing to a Terms of Service page (optional, default is none):
 ;termsOfServiceUrl = "https://something"
+
+[Sorting]
+; Sorting in VuFind is case insensitive and does not respect locales. By setting
+; this option to true, the sorting will be respecting locales but also case
```

Comment:
Is there a reason for case-sensitivity? Could we rather set the strength by default to `Collator::SECONDARY` or make it configurable?